1-react/components/timer.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { connect } from '../store/slomux'
+
+class IntervalComponent extends React.Component {
+  render() {
+    return (
+      <div>
+        <span>Интервал обновления секундомера: {this.props.currentInterval} сек.</span>
+        <span>
+          <button onClick={() => this.props.changeInterval(-1)}>-</button>
+          <button onClick={() => this.props.changeInterval(1)}>+</button>
+        </span>
+      </div>
+    )
+  }
+}
+
+const Interval = connect(dispatch => ({
+  changeInterval: value => dispatch(changeInterval(value)),
+}),
+state => ({
+  currentInterval: state,
+}))(IntervalComponent)
+
+class TimerComponent extends React.Component {
+  state = {
+    currentTime: 0
+  }
+
+  render() {
+    return (
+      <div>
+        <Interval />
+        <div>
+          Секундомер: {this.state.currentTime} сек.
+        </div>
+        <div>
+          <button onClick={this.handleStart}>Старт</button>
+          <button onClick={this.handleStop}>Стоп</button>
+        </div>
+      </div>
+    )
+  }
+
+  handleStart() {
+    setTimeout(() => this.setState({
+      currentTime: this.state.currentTime + this.props.currentInterval,
+    }), this.props.currentInterval)
```

Comment:
`this.props.currentInterval` это количество секунд? `setTimeout/setInterval` принимают время в миллисекундах.

Предлагаю сделать конвертацию:
```suggestion
    }), this.props.currentInterval * 1000)
```